script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_10.txt

```diff
@@ -2,7 +2,7 @@
 "I'm exhausted."
 I roll over onto my back.
 My heart and lungs scream with joy.
-My breathing, which has been stopped for so long, finally resumes. // dropping the 'like anyone else nuance. 人並みに再開させた
+My breathing, which has been stopped for so long, finally resumes.// dropping the 'like anyone else nuance. 人並みに再開させた
```

Comment:
I wonder if this could be rephrased as 
It's as if I can finally start breathing again.

Reply:
"My breathing, which feels like it's been paused forever, finally resumes"?

Reply:
> 長く止められていた呼吸を、ようやく、人並みに再開させた

I think it needs to be directly mentioned that his breathing had stopped for a long time